rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -189,6 +191,7 @@ public LRAData getLRAInfo(
     @POST
     @Path("start")
     @Produces(MediaType.TEXT_PLAIN)
+    @Bulkhead
```

Comment:
I'm interested what is the reason for taking the faultolerance `@Bulkhead` on start method.
Having the fault tolerance capability is probably fine but why not other options (e.g. `@Retry`)? Why we want to work with only 10 parallel requests - what is the reason? Are we fine in adding one more dependency to the mix for the LRA processing?

Reply:
Because we want to protect the coordinator against overload. This PR is to address a particular issue I found during my stress testing of the coordinator (the issue really needs to be a blocker). Throttling start requests will reduce the number of active LRAs that the coordinator manages. So applying a bulkhead to start will reduce the total number of parallel LRAs to a manageable number and the other coordinator requests will all apply to this reduced number of LRAs.

I chose the default because, well it is the default. The user may then configure a value that is appropriate to his installation (using the usual MP FT config mechanism).

I didn't need any of the other FT annotations because the intent of the PR is to protect the coordinator. In the PR description I did explain why we don't use most of the FT features:

> There is a lot we can do in this area [fault resilience] and we have a few JIRAs for tackling the various requirements. But this PR is a basic minimum to reduce the number of errors caused by overloading the system in which LRA operates.

Reply:
Generally, this PR looks good. But I just want to point out that `@Bulkhead` is only reducing concurrent executions of the method thus at max 10 thread can run `start` at the same time but that doesn't correspond to the reduction of the number of active LRAs that the coordinator manages. The errornous state can still be reached but it would take a few retries on the /start calls.

Reply:
@mmusgrov sure, I was not aware about the problem from the stress testing. Then this makes sense. Thanks.

Reply:
I ran a local test to ensure that the Bulkhead was being respected.